stat/distuv/logistic_test.go

```diff
@@ -0,0 +1,115 @@
+package distuv
+
+import (
+	"math"
+	"testing"
+)
+
+func TestLogisticParameters(t *testing.T) {
+	l := Logistic{Mu: 0, S: 0}
+
+	if l.NumParameters() != 2 {
+		t.Fail()
+	}
+
+	if l.ExKurtosis() != 6/5 {
+		t.Fail()
+	}
+
+	if l.Skewness() != 0 {
+		t.Fail()
+	}
+
+	if l.Mean() != l.Mu {
+		t.Fail()
+	}
+
+	if l.Mean() != l.Median() {
+		t.Fail()
+	}
+
+	if l.Mean() != l.Mode() {
+		t.Fail()
+	}
+}
+
+func TestLogisticStdDev(t *testing.T) {
+	const sq3 = 1.732050807568877293527446341505872366942805253810380 // sqrt(3)
+
+	l := Logistic{Mu: 0, S: sq3 / math.Pi}
+	if l.StdDev() != 1 {
+		t.Fail()
+	}
+
+	if l.Variance() != 1 {
+		t.Fail()
+	}
+}
+
+func TestLogisticCDF(t *testing.T) {
```

Comment:
Would be nice to see some tests for PDF and CDF for some general values of parameters and argument, comparing the result with e.g. value returned by a reference implementation (scipy? Mathematica? Matlab?).

Reply:
Added couple of test values from WolframAlpha